src/App.js

```diff
@@ -43,6 +43,8 @@ import namehash from 'eth-ens-namehash'
 //https://github.com/lesnitsky/react-native-webview-messaging/blob/v1/examples/react-native/web/index.js
 import RNMessageChannel from 'react-native-webview-messaging';
 
+import YourModule from './components/YourModule'
```

Comment:
As far as I understood, that module is concerned with fetching NFTs from Opensea.io. I'd suggest we give it a more proper name that describes what it does.